columnar/src/compat_tests.rs

```diff
@@ -0,0 +1,84 @@
+use crate::{Column, ColumnarReader, DynamicColumn, CURRENT_VERSION};
+
+const NUM_DOCS: u32 = u16::MAX as u32;
+
+fn generate_columnar(num_docs: u32) -> Vec<u8> {
+    use crate::ColumnarWriter;
+
+    let mut columnar_writer = ColumnarWriter::default();
+
+    for i in 0..num_docs {
+        if i % 100 == 0 {
+            columnar_writer.record_numerical(i, "sparse", i as u64);
+        }
+        if i % 2 == 0 {
+            columnar_writer.record_numerical(i, "dense", i as u64);
+        }
+        columnar_writer.record_numerical(i, "full", i as u64);
+        columnar_writer.record_numerical(i, "multi", i as u64);
+        columnar_writer.record_numerical(i, "multi", i as u64);
+    }
+
+    let mut wrt: Vec<u8> = Vec::new();
+    columnar_writer.serialize(num_docs, None, &mut wrt).unwrap();
+
+    wrt
+}
+
+#[test]
+#[ignore]
+/// Unignore to write columnar for current version to disk
+fn create_format() {
```

Comment:
I'm not fond of the #[ignore] trick.. What about running it all of the time, but skip the generation if the file already exists for instance?